docs/src/examples/components/Avatar/Variations/AvatarExampleSize.shorthand.tsx

```diff
@@ -5,15 +5,22 @@ import { Avatar } from '@stardust-ui/react'
 const AvatarExampleSizeShorthand = () =>
   _.times(10, i => {
     const size = i + 1
-
     return (
-      <div style={{ background: 'white' }}>
-        <Avatar
-          key={size}
-          size={size}
-          src="/public/images/avatar/small/matt.jpg"
-          status="Available"
-        />
+      <div style={{ background: 'white', marginBottom: '10px' }}>
+        <div style={{ background: 'inherit', width: '10%', float: 'left', marginBottom: '10px' }}>
+          <Avatar
+            key={size}
+            size={size}
+            src="public/images/avatar/small/matt.jpg"
+            status="Available"
+          />
+        </div>
+        <div style={{ background: 'inherit', width: '10%', float: 'left', marginBottom: '10px' }}>
+          <Avatar key={size * 100} size={size} name="John Doe" status="Available" />
+        </div>
+        <div style={{ background: 'inherit', width: '80%', float: 'left', marginBottom: '10px' }}>
+          <Avatar key={size * 1000} size={size} src="public/images/avatar/small/matt.jpg" />
```

Comment:
Note, `key` is only required by elements at the root of the array.  Elements that are JSX children do not require a `key`.

The `key` can be removed from all `Avatar` components and instead a single `key` placed on the root `div`.